gui/src/maths/vector3.ts

```diff
@@ -3,6 +3,12 @@ import { Vector3 } from 'three';
 
 export type Vector3Object = { x: number; y: number; z: number };
 
+export function averageVector(vecs: Vector3[]) {
+  if (vecs.length === 0) return new Vector3();
+  const sum = vecs.reduce((prev, curr) => prev.add(curr), new Vector3());
+  return sum.divideScalar(vecs.length);
```

Comment:
This seems like a solid chance of having a length of 0, which would not be good to divide by